geopandas/io/tests/test_sql.py

```diff
@@ -181,3 +271,333 @@ def test_read_postgis_binary(self, connection_spatialite, df_nybb):
         )
         df = read_postgis(sql, con, geom_col=geom_col)
         validate_boro_df(df)
+
+    def test_write_postgis_default(self, df_nybb):
+        """Tests that GeoDataFrame can be written to PostGIS with defaults."""
+        engine = connect_engine("test_geopandas")
+        if engine is None:
+            raise pytest.skip()
```

Comment:
If we turn `connect_engine` into a fixture (like I did for `connection_postgis` now), then we can get rid of the skipping here, and also the `try/finally: engine.dispose()` below can be handled inside the fixture instead of needing to repeat here in every test.

Reply:
Created now a fixture for engine as well and updated accordingly.